Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -622,7 +622,7 @@ def genS(fragment,howMuch):
 
 ### Fastsim: template to produce signal and overlay it with premixed minbias events
 FS_PREMIXUP15_PU25_OVERLAY = merge([
-        {"-s" : "GEN,SIM,RECOBEFMIX,DIGIPREMIX_S2:pdigi_valid,DATAMIX,L1,L1Reco,RECO,HLT:@relval25ns,VALIDATION",
+        {"-s" : "GEN,SIM,RECOBEFMIX,DIGIPREMIX_S2:pdigi_valid,DATAMIX,L1,L1Reco,RECO,HLT:@relval2016,VALIDATION",
```

Comment:
No, I really mean DIGI2RAW - HLT expects RAW (for fast sim at least L1 packed up in the RAW) and unpacks it itself.